frontend/nexus/user.ts

```diff
@@ -85,6 +85,28 @@ schema.extendType({
       },
     })
 
+    t.field('updateUser', {
+      type: 'User',
+      args: {
+        userId: schema.intArg({ required: true }),
+        handle: schema.stringArg({ required: false }),
+        email: schema.stringArg({ required: false }),
+        name: schema.stringArg({ required: false }),
+        profileImage: schema.stringArg({ required: false }),
+      },
+      resolve: async (_parent, args, ctx: any) => {
+        const updates = { ...args }
+        delete updates.userId
+
+        return ctx.db.user.update({
```

Comment:
This can fail since `handle` has the unique constraint. I'm not sure we want to allow users to change their handles, but if we really do then we should handle collisions more gracefully and notify users on the FE if their target handle is taken.

Reply:
Great point! I do want people to be able to change their handle if they would like to at some point but I think it would be best to simply disable this field for now so we can work on a nice experience for checking handle availability